tests/common/test_utils.py

```diff
@@ -73,7 +73,7 @@ def construct_measurement(model_name, gpu_metric_values, non_gpu_metric_values,
 
     # Perf Config needs a protocol
     perf_config = PerfAnalyzerConfig()
-    perf_config['model-name'] = model_name
+    perf_config['model-names'] = model_name
```

Comment:
can `construct_measurement()` also take a comma-separated model name list string, now? instead of just a single model name string?

Reply:
This is just a single string (that may have multiple model names separated by strings). This is just a test helper function

Part of the confusion was perf_config_parameters[model-names] vs PerfConfig[model-names]. The first needs an array of possible values. The second is a string.